pkg/controllers/webhook/controller.go

```diff
@@ -689,6 +689,26 @@ func (c *controller) buildDefaultResourceValidatingWebhookConfiguration(cfg conf
 				SideEffects:             sideEffects,
 				AdmissionReviewVersions: []string{"v1"},
 				TimeoutSeconds:          &c.defaultTimeout,
+			}, {
+				Name:         config.ValidatingWebhookName + "-fail",
```

Comment:
What about `buildDefaultResourceMutatingWebhookConfiguration`? I don't see this for mutate either.

Reply:
Do we need it too ? I was not sure, going to add it there too.

Reply:
done

Reply:
I don't know, would be good to be consistent with validate:)